src/pixie/demo.nim

```diff
@@ -1,10 +1,6 @@
-import opengl, pixie, pixie/context
+import opengl, pixie, pixie/context, staticglfw except Image
```

Comment:
nimpretty issue

Reply:
Hm, for some reason I thought you mentioned you'd fixed morepretty. Did i miss-remember or is it not pushed?

Reply:
Ah i bet i needed to nimble install not just nimble build, never can remember with these